instrumentation/opentelemetry-instrumentation-asyncpg/src/opentelemetry/instrumentation/asyncpg/__init__.py

```diff
@@ -150,8 +150,8 @@ def _uninstrument(self, **__):
 
     async def _do_execute(self, func, instance, args, kwargs):
         exception = None
-        params = getattr(instance, "_params", {})
-        name = args[0] if args[0] else params.get("database", "postgresql")
+        params = getattr(instance, "_params")
+        name = args[0] if args[0] else getattr(params, "database", "postgresql")
```

Comment:
Nice. Could this also be updated in [_do_cursor_execute](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/317f7a6150d73f89959e8ec44c6d9f599977d289/instrumentation/opentelemetry-instrumentation-asyncpg/src/opentelemetry/instrumentation/asyncpg/__init__.py#L189-L193)?

Reply:
Done!